src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.search {
+  display: flex;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+}
+
+.search--big {
+  height: 70px;
+}
+
+.search--small {
+  height: 42px;
+}
+
+.search__label {
+  flex-shrink: 0;
+  background: url(./images/Search.svg) no-repeat center/contain;
+}
+
+.search__label--big {
+  flex-basis: 19px;
+  margin-left: 26px;
+}
+
+.search__label--small {
+  flex-basis: 11px;
+  margin-left: 13px;
+}
+
+.search__input {
+  width: 100%;
+  border: none;
+  font-family: "Avenir", sans-serif;
+}
+
+.search__input--big {
+  padding-left: 17px;
+  font-size: 16px;
+}
+
+.search__input--small {
+  padding-left: 9px;
+  font-size: 14px;
+}
+
+.search__input:focus {
+  outline: none;
+}
+
+.search:focus-within {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
gradient is visible only on the small part of the input, biggest part (actually `input`) is white on focus:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/e1049908-a796-4889-abc0-4eb5a78e3a73)